frontends/web/src/locales/en/app.json

```diff
@@ -428,23 +428,23 @@
         },
         "pocket": {
           "fees": {
-            "info": "Bank transfer: {{fee}}%",
+            "info": "{{fee}}% exchange fee, plus small network transaction fee",
             "title": "Fees"
           },
           "learnMore": "Learn more about Pocket",
           "payment": {
             "bankTransfer": "Bank transfer",
             "bankTransferDetails": {
-              "sepa": "SEPA and SEPA Instant (EUR transactions in SEPA countries only)",
+              "sepa": "SEPA and SEPA Instant (EUR)",
               "sic": "Swiss Interbank Clearing (CHF transactions in CH/LI only)",
-              "uk": "UK Faster Payments (GBP transactions in the UK only)"
+              "uk": "UK Faster Payments (GBP)"
             },
             "bankTransferReccuring": "How to set up recurring purchases with a standing order?",
             "title": "Payment methods"
           },
           "supportedCurrencies": "Supports European currencies: EUR, GBP, and CHF.",
           "verification": {
-            "info": "Only requires identity verification above daily and annual thresholds.",
+            "info": "No identitiy verification for amounts below threshold (~950 EUR/day)",
```

Comment:
These 4 keys need to be updated manually on locize once `staging-pocketsell` is merged to `master`.

Reply:
Do we want to keep this commit anyway? Also, I think it still misses the section to say that to sell you need to have bought with pocket before, as @jadzeidan suggested. Maybe he can draft a wording for that?

Reply:
Will wait for the draft 👍 thanks.